src/sagas.js

```diff
@@ -38,33 +44,39 @@ function parseData(raw) {
   const fulfillmentsAcceptedCum = [[], []];
   const bountiesIssuedCum = [[], []];
 
-  for (let i = 0; i < raw.length; i += 1) {
-    const date = Date.parse(raw[i].date);
-    const p = raw[i].is_week ? 1 : 0;
+  for (let i = 0; i < categoryData.length; i += 1) {
+    categories.push([categoryData[i].prioritized_name, categoryData[i].total_count]);
```

Comment:
this works - I tend to actually like for i = 0 loops - you can make them the most efficient. :) Also could use map or reduce functions... but I'd say don't worry about it. This is clean as well.

Reply:
😀